1-introduction/standard_lib_Task1.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+
+l = [1, 2, '3', 4, None, 10, 33, 'Python', -37.5]
+
+
+def filter_list(l):
+
+    # •	“for” loop
+    def for_filter_list():
+        for i in l:
+            if type(i) is not int:
+                l.remove(i)
```

Comment:
You should not use remove and modify the original list

Reply:
This is fixed - thanks!